src/Cli/dotnet/commands/InstallingWorkloadCommand.cs

```diff
@@ -242,6 +243,20 @@ protected void UpdateWorkloadManifests(WorkloadHistoryRecorder recorder, ITransa
                 return;
             }
 
+            if (resolvedWorkloadSetVersion?.Contains('@') == true)
```

Comment:
For `install` and `update`, it's the workload ID argument that should be used to specify the version.  For example, `dotnet workload install ios@17.5.9231-net9-p7 tvos@17.5.9232-net9-p7`.  Note that the different workloads are separated by a space here, not a semicolon.

Reply:
This doesn't align with the guidance marcpopMSFT provided:
> We decided to just use the existing --version or workloadVersion in the global.json for this and look for the @ symbol.

https://github.com/dotnet/sdk/issues/42367#issuecomment-2489867789

Did I miss a meeting on this in which we changed the plan?